### ISSUE TITLE
Multiple elements with non-unique id

### ISSUE THREAD

**Issue by HolgerJeromin** [openstreetmap/openstreetmap-website#2520]
Chrome complains:

![image](https://user-images.githubusercontent.com/2410353/73133352-a5ee9500-4027-11ea-9bf9-c4922d9d6eaa.png)

This DOM feels not right indeed:
![image](https://user-images.githubusercontent.com/2410353/73133348-94a58880-4027-11ea-9030-83e5cf6e389e.png)

**Comment by tomhughes** [comment]
This is useless unless you tell us what page you are referring to.

**Comment by HolgerJeromin** [comment]
As this is the main page `www.openstreetmap.org` with most subpages with map (way, node, relation) I forgot to mention :)

**Comment by tomhughes** [comment]
I think this is because there are two different versions of the form, one for the normal view and one for the mobile view, so it's probably not very easy to fix.

I suspect nothing actually uses the ids but rails probably generates them automatically - actual usage is probably via the form field names and hopefully always uses the form that is actually live and visible.

**Comment by mmd-osm** [comment]
This seems to be related to partial  `app/views/layouts/_search.html.erb` being included twice in app/views/layouts/map.html.erb:

```
<% content_for :header do %>
  <%= render :partial => "layouts/search", :locals => { :autofocus => false } %>
<% end %>

<% content_for :content do %>
  <div id="sidebar">
    <%= render :partial => "layouts/search", :locals => { :autofocus => true } %>

```

Header level form was originally (re-)introduced in https://github.com/openstreetmap/openstreetmap-website/commit/dd12e1de637d5242c8ac7ed933fd8a3f70a70367

**Comment by tomhughes** [comment]
Isn't that exactly what I said?

The first one is for the mobile site and the second one for the desktop site.

**Comment by mmd-osm** [comment]
Yes, you've already mentioned that. I thought it might be interesting to have a bit of context where this happens in the codebase.
